process/transaction/metaProcess.go

```diff
@@ -71,8 +71,11 @@ func NewMetaTxProcessor(args ArgsNewMetaTxProcessor) (*metaTxProcessor, error) {
 		marshalizer:             args.Marshalizer,
 		scProcessor:             args.ScProcessor,
 		flagPenalizedTooMuchGas: atomic.Flag{},
+		shouldCheckBalanceHandler: func() bool {
```

Comment:
indent. There is no balance check in metachain.

Reply:
there are no accounts in meta.

Reply:
removed this.